docs/src/main/tut/json.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: JSON handling
+---
+
+## Pick a library
+
+### Argonaut
+
+```scala
+libraryDependencies += Seq(
+  "org.http4s" %% "http4s-argonaut" % "0.15.0a-SNAPSHOT",
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.1" % "1.1.1" // or 1.1.0 for non-a versions
```

Comment:
I like the pointer for non-a users.

It might be nice to explain why this non-http4s dep is here (deriviation).  Also, it's akin to circe-generic.  We should mention both or neither.